features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -694,6 +694,7 @@
     ]
   },
   "console.attribute_dialects.enabled": true,
+  "console.ui.isDefaultDialectEditingEnabled": false,
```

Comment:
The convension here is to have word separators by _

Reply:
Also is the "ui" part required?

Reply:
https://github.com/wso2/product-is/issues/11166  issue created @brionmario @pulasthi7